src/capitalsFirst.js

```diff
@@ -18,7 +18,19 @@
  * @returns {string} - string with uppercase words in front
  */
 function capitalsFirst(str) {
-  // write code here
+  let initialArray = str.split(' ');
+  let uppercaseLetterFirst = [];
+  let lowercaseLetterFirst = [];
+
+  for (let i = 0; i < initialArray.length; i++) {
+    if (initialArray[i][0] >= 'A' && initialArray[i][0] <= 'Z') {
```

Comment:
If you use the same array/object property more then once -> move it to constant
(About initialArray[i], initialArray[i][0])